python-sdk/src/astro/databases/base.py

```diff
@@ -455,6 +455,17 @@ def load_file_to_table(
                 chunk_size=chunk_size,
             )
 
+    @staticmethod
+    def get_dataframe_from_file(file: File):
+        """Get pandas dataframe file
+        :param file: File path and conn_id for object stores
+        """
+        # We need export_to_dataframe() for Biqqery, Snowflake and Redshift except for Postgres. For postgres
+        # we are overriding this method and using export_to_dataframe_via_byte_stream(). export_to_dataframe_via_
+        # byte_stream copies files in a buffer and then use that buffer to ingest data. With this approach
+        # we have significant performance boost for postgres.
+        return file.export_to_dataframe()
```

Comment:
@utkarsharma2 why not put this into the function description? Seems like a good explanation for why this function exists

Reply:
@dimberman Done.